homeassistant/components/wake_on_lan/const.py

```diff
@@ -1,3 +1,11 @@
 """Constants for the Wake-On-LAN component."""
 
+from homeassistant.const import Platform
+
 DOMAIN = "wake_on_lan"
+PLATFORMS = [Platform.BUTTON]
+
+CONF_OFF_ACTION = "turn_off"
```

Comment:
The turn off action constant didn't need to move since it's only used in one module.